lib/components/home/token_view.dart

```diff
@@ -90,7 +96,7 @@ class _TokenViewState extends State<TokenView> {
                         Observer(builder: (context) {
                           return BlurableContent(
                               Text(
-                                "${balance != null && balance > BigInt.zero ? toAmountDisplayBigInt(balance, fractionDigits: 2) : 0} ${tokenName != "" ? tokenName : name.toUpperCase()}",
+                                "${balance != null && balance > BigInt.zero ? NumberFormat.compact().format((balance + BigInt.from(1003000)) / BigInt.from(10).pow(18)).toString() : 0} ${tokenName != "" ? tokenName : name.toUpperCase()}",
```

Comment:
Why is 1003000 hardcoded?

Reply:
This is a typo I forgot to remove I'll correct this ASAP